intrahospital_api/deployment_check.py

```diff
@@ -0,0 +1,201 @@
+"""
+Gets called by the deployment check management command.
+
+check_since is the main method
+
+Takes a datetime and looks at all patients who don't have archived tags.
+
+Looks at all lab tests that have a last_updated after the datetime for those patients.
+
+Then deletes them and runs a batch load since that time. Looks again.
+
+Then deletes them and runs initial patient loads for all patients. Looks again.
+
+The rolls back all changes in a transaction.
+
+It populates a result dict with
+current: The current state of the db.
```

Comment:
Can we be more explicit than 'current state' ? e.g. I don't know what that means from this text, I have to read it, run the code, or both to find out...